lte/gateway/Makefile

```diff
@@ -75,21 +75,18 @@ TEST_FLAG = -DBUILD_TESTS=1
 
 all: build
 
```

Comment:
Are these intentional changes?

Reply:
There was a proto dependeny.

Reply:
No I mean the make command changes

Reply:
@gurrapualt can we add the envoy_controller related commands back in?

Reply:
yes added.